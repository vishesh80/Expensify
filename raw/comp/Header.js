import React from 'react';
import {NavLink} from 'react-router-dom';

const Header = () => (<div>
    <h1>Expense Tracker</h1>
    <NavLink to="/" exact={true}>Dashboard</NavLink>
    <NavLink to="/create" exact={true}>Create Expense</NavLink>
    <NavLink to="/help" exact={true}>Help</NavLink>
</div>);

export default Header;