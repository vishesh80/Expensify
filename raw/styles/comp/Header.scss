.header
{
    width: 100%;
    min-width: 400px;
    max-width: 1500px;
    margin: auto;
    background-color: rgb(14, 39, 71);
    padding: 2em 3em;
    display: flex;
    justify-content: space-between;
    align-items: center;

    .navlink
    {
        text-decoration: none;
        color: white !important;
        font-size: 2.5rem;
    }

    button
    {
        background-color: rgb(32, 166, 255);
        font-size: 1.1rem;
        padding: .4em .8em;
        border: none;
        color: white;
    }

}