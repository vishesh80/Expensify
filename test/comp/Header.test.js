import {shallow} from 'enzyme';
import React from 'react';
import Header from '../../raw/comp/Header';
import eToJ from 'enzyme-to-json';

test('Header component test',() => {

    let wrapper = shallow(<Header/>);
    expect(eToJ(wrapper)).toMatchSnapshot();
});


