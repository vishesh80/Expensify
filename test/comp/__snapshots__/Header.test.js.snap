// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component test 1`] = `
<div>
  <h1>
    Expense Tracker
  </h1>
  <[object Object]
    exact={true}
    to="/"
  >
    Dashboard
  </[object Object]>
  <[object Object]
    exact={true}
    to="/create"
  >
    Create Expense
  </[object Object]>
  <[object Object]
    exact={true}
    to="/help"
  >
    Help
  </[object Object]>
</div>
`;
